LamaDescriptorIdentifier id 
---
float64[] descriptor 
